guides/common/assembly_troubleshoot-dnf-modules.adoc

```diff
@@ -0,0 +1,8 @@
+[id="troubleshoot-dnf-modules_{context}"]
+
+include::modules/con_troubleshooting-dnf-modules.adoc[]
+include::modules/con_troubleshooting-ruby.adoc[leveloffset+=1]
+
+ifdef::katello,orcharhino,satellite[]
```

Comment:
This is incorrect because you lose the section on `foreman-el` for `Installing Server`, where it is relevant.

Put in a different way. For Installing Server the table is:

| | foreman-el | foreman-deb | katello |
| - | - | - | - |
| DNF Ruby | x | - | x |
| DNF PostgreSQL | x | - | x |

For Installing Proxy:

| | foreman-el | foreman-deb | katello |
| - | - | - | - |
| DNF Ruby | x | - | x |
| DNF PostgreSQL | - | - | x |

Reply:
Addressed your both views.